### DISCUSSION TITLE
是否有计划开发 IOS、安卓、windows、MacOS 客户端版本

### DISCUSSION THREAD

**Discussion by feodorren** [AlistGo/alist#1207] [Ideas]
理论上Alist是一个接口，然后缓存在本地，或者刷新缓存，那么根据目前电脑或者手机的性能，可否考虑移植到客户端设备？毕竟Alist还是需要远程服务器，或者NAS等设备。

而且Alist的配置多端同步也非常方便，毕竟备份就是一个```.conf```文件。

**Comment by xiaowang7212** [comment]
作者：毫无必要

**Reply by Dashan-IZ** [reply]
毫无必要

**Comment by Godfox9527** [comment]
安卓通过第三方可以实现吧，我倒是希望ios能支持，我只有ios备用机，没有客户端太可惜了

**Reply by Code2qing** [reply]
ish可以试试，不过是转译为x86指令的，性能超级辣鸡

**Reply by MADAO-NW** [reply]
> ish可以试试，不过是转译为x86指令的，性能超级辣鸡
大佬ish怎么实现的啊，用的哪个发行版本啊

**Comment by 12189108** [comment]
MacOS用docker安装,安卓用termux运行，windows有对应版本

**Reply by Yefori-Go** [reply]
MacOS直接下载darwin平台的包就行了吧

**Comment by evlic** [comment]
有没有可能你说的是服务端……？

alist 客户端是 web 全平台通用啊

**Comment by Dashan-IZ** [comment]
我倒是希望alist能把网页预览的视频播放器和音乐播放器能重新写一下

**Comment by ypq123456789** [comment]
安卓服务端github上有的

**Reply by kkgit2008** [reply]
只有tar.gz文件，请教一下，这个文件具体要怎么使用

**Comment by sunzongzheng** [comment]
ios服务端：https://github.com/sunzongzheng/alist-expo

**Comment by ypq123456789** [comment]
搜 alistflutter



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年7月14日(周日) 中午1:37
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [alist-org/alist] 是否有计划开发 IOS、安卓、windows、MacOS 客户端版本 (Discussion #1207)




 
只有tar.gz文件，请教一下，这个文件具体要怎么使用
